spec/section_spec.rb

```diff
@@ -85,5 +86,26 @@ class Page < SitePrism::Page
       expect(section).to respond_to :execute_script
       expect(section).to respond_to :evaluate_script
     end
+
+    it 'responds to Capybara::Node::Element methods' do
+      expect(section).to respond_to :visible?
+      expect(section).to respond_to :text
```

Comment:
I appreciate this is different from other areas of the code-base, But I'm gonna start getting a style into this suite slowly. Can we separate these expectations